client/pages/index.js

```diff
@@ -1,78 +1,128 @@
-import React from 'react'
-import Header from "@components/Header";
-import LayoutStack, { Line, Spacing } from '@components/Layout'
-import { FlexWrapper, InputWrapper } from '@components/chakra'
-import { SmallCard } from '@components/cards'
-import { HeadingTwo, Paragraph } from '@components/Typography'
-
+import {
+  Box,
+  Button,
+  Container,
+  Divider,
+  Flex,
+  Heading,
+  Text,
+} from '@chakra-ui/core';
+import CountryCard from '@components/Cards/CountryCard';
+import PeopleCard from '@components/Cards/PeopleCard';
+import Search from '@components/Searchbar';
+import RoundedIcon from '@components/Utils/RoundedIcon';
+import Axios from 'axios';
+import { ArrowLeft, ArrowRight } from 'theme/customIcons';
 
 export default function Home() {
-  return (
-    <React.Fragment>
+  const [data, setData] = React.useState([]);
 
-      <Header />
-      <LayoutStack>
-        <section>
-          <Spacing />
-          <HeadingTwo>Know your Statesman</HeadingTwo>
-          <Paragraph><bold>400</bold> Publicly available Manifestos of your elected politicians and statesmen</Paragraph>
-          <InputWrapper width="60%" placeholder="search" />
-          <select placeholder="Country"><option>Ghana</option></select>
-          <Spacing />
-        </section>
-        <Line />
+  React.useEffect(() => {
+    const fetchData = async () => {
```

Comment:
Can we have like a `resource.js` or something similar for API so we don't directly call the api like in this place we'd rather do 

```jsx
import {COUNTRY_RESOURCE} from 'path'

Axios.get(COUNTRY_RESOURCE)
```

Feel free to use a naming convention you deem fit

Reply:
Okay sure, I will work on that and create a PR for it tomorrow

Reply:
Meanwhile we need to work on your PR Titles :laughing:

Reply:
Haha, that's something, I will work on before my next PR :)